frontend/src/layout/lemonade/TopBar/index.scss

```diff
@@ -132,6 +132,22 @@
     cursor: pointer;
 }
 
+.SitePopover__profile-picture {
```

Comment:
Nit: I believe we aligned on a convention of calling scss and ts files based on their actual name and avoid `index.scss` / `index.ts` as it makes it hard to lookup from the IDE.

Also, perhaps worth using SCSS nested classes instead?

Reply:
Yeah, I agree, I will make those refactors in a more targeted PR.